profiles/apparmor/apparmor.go

```diff
@@ -66,25 +66,29 @@ func macroExists(m string) bool {
 // InstallDefault generates a default profile in a temp directory determined by
 // os.TempDir(), then loads the profile into the kernel using 'apparmor_parser'.
 func InstallDefault(name string) error {
-	p := profileData{
-		Name: name,
-	}
+	var (
+		cmd  *exec.Cmd
+		err  error
+		pipe io.WriteCloser
+		p    profileData
+	)
+
+	p.Name = name
+	cmd = exec.Command("apparmor_parser", "-Kr")
```

Comment:
Do you mean the execution-related code or more? I initially worked on consolidating all AppArmor code (i.e., pkg/ and profiles/), but I was afraid to break dependencies and decided to come up with a less dramatic change.